osx-manual-setup.md

```diff
@@ -65,60 +64,78 @@ xcode-select -—install
 ### Dependencies
 
 ```
-brew install chromedriver imagemagick@6 gs pkg-config openssl geckodriver sphinx
+brew install chromedriver imagemagick@6 gs pkg-config openssl geckodriver sphinx mysql@5.7 postgresql@14
 brew link imagemagick@6
 ```
 
-* **Macs with M1** also require de following:
+* **Macs with M1** also require the following:
```

Comment:
will you include heare the instructions to install `local-fastimage_resize`?
https://github.com/3scale/porta/pull/3065#issuecomment-1447900185

Reply:
Once the PR is verified and merged, yes